packages/api-clients/open-api/authorizationApi.yml

```diff
@@ -1348,7 +1347,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/ProducerKeychain"
```

Comment:
For consistency with clients, I would also change the return type of `addProducerKeychainEService` from 204 to 200 returning a FullProducerKeychain - same as happening in `addClientPurpose` (the corresponding operation for clients)